WordPress/src/main/java/org/wordpress/android/ui/people/utils/PeopleUtils.java

```diff
@@ -46,8 +49,13 @@ public void onErrorResponse(VolleyError volleyError) {
             }
         };
 
+        Map<String, String> params = new HashMap<>();
+        params.put("number", Integer.toString(PeopleUtils.FETCH_USERS_LIMIT));
+        params.put("offset", Integer.toString(offset));
+        params.put("order_by", "display_name");
```

Comment:
Although [the docs](https://developer.wordpress.com/docs/api/1.1/get/sites/%24site/users/) do say `display_name` is a valid `order_by` parameter, I don't see `display_name` in the response. Are we sure this is correct?